code/controllers/communications.dm

```diff
@@ -205,9 +205,13 @@ var/const/RADIO_MAGNETS = "radio_magnet"
 var/const/RADIO_LOGIC = "radio_logic"
 
 var/global/datum/controller/radio/radio_controller
+var/global/list/headsets[0]
```

Comment:
I know, just matching the weird convention already established for the other global lists.